arbnode/inbox_tracker.go

```diff
@@ -606,8 +606,11 @@ func (t *InboxTracker) AddSequencerBatches(ctx context.Context, client arbutil.L
 		ctx:    ctx,
 		client: client,
 	}
-
-	multiplexer := arbstate.NewInboxMultiplexer(backend, prevbatchmeta.DelayedMessageCount, t.das, t.blobReader, arbstate.KeysetValidate)
+	daProviders := []arbstate.DataAvailabilityProvider{
+		arbstate.DAProviderWrapperDAS(t.das),
+		arbstate.DAProviderWrapperBlobReader(t.blobReader),
```

Comment:
Does this end up creating the wrappers even if the underlying reader is nil, and then causing issues with the nil checks later?

Reply:
fixed it.